Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            bool calculateAgain = true;
+
+            while (calculateAgain)
+            {
+                Console.WriteLine("BMI Calculator");
+
+                string name = GetString("Enter your name: ");
+                int age = GetInt("Enter your age: ");
+                float weight = GetFloat("Enter your weight(kg): ");
+                float height = GetFloat("Enter your height(cm): ");
+
+                Console.WriteLine(name + " is " + age + " years old, his weight is " + weight + " kg and his height is " + height + " cm.");
+
+                float BMI = CalculateBMI(weight, height / 100);
+
+                Console.WriteLine("Your BMI is " + BMI);
+
+                WeightType(BMI);                
+
+                Console.WriteLine("Do you want to calculate another persoon's BMI? (y/n)");
+                string continueTheFight = Console.ReadLine();
+                while (continueTheFight != "y" && continueTheFight != "n")
+                {
+                    Console.WriteLine("That is not a valid response.");
+                    Console.WriteLine("Do you want to calculate another persoon's BMI? (y/n)");
+                    continueTheFight = Console.ReadLine();
+                }
+
+                if (continueTheFight == "n")
+                {
+                    calculateAgain = false;
+                }
+
+            }
+
+            Console.WriteLine("Exiting...");
+                        
+        }
+                
+        public static string GetString(string message)
+        {
+            Console.Write(message);
+            return Console.ReadLine();
+        }
+
+        public static int GetInt(string message)
+        {
+            Console.Write(message);
+            return Convert.ToInt32(Console.ReadLine());
+        }
+
+        public static float GetFloat(string message)
+        {
+            Console.Write(message);
+            return Convert.ToSingle(Console.ReadLine());
+        }
+
+        public static float CalculateBMI(float weight, float height)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename method 'CalculateBMI' to match pascal case naming rules, consider using 'CalculateBmi'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4863604)